sdk/openai/Azure.AI.OpenAI.Assistants/src/Generated/UploadFileRequest.Serialization.cs

```diff
@@ -27,7 +28,14 @@ void IJsonModel<UploadFileRequest>.Write(Utf8JsonWriter writer, ModelReaderWrite
 
             writer.WriteStartObject();
             writer.WritePropertyName("file"u8);
-            writer.WriteBase64StringValue(Data.ToArray(), "D");
+#if NET6_0_OR_GREATER
+				writer.WriteRawValue(global::System.BinaryData.FromStream(Data));
```

Comment:
Why is this prefixed with `global`?

Reply:
well this is a shortage of our current generator.
The generator always writes the fully qualified name of a type for all cases, and when all the code is generated, we will do a roslyn format and reduce, which trims off unnecessary parts, such as `global::` and the namespaces exist in usings.
But this is special, because this is in a preprocessor, the content Roslyn gets does not have this part, therefore this is not cleaned up by it.
We currently do not have a solution for this. Considering this works just fine, I think we could let it be.